### DISCUSSION TITLE
Welcome to CDT Discussions!

### DISCUSSION THREAD

**Discussion by artem-ogre** [artem-ogre/CDT#28] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by RotateMe** [comment]
Hi, first of all thanks for all the effort and for providing an extremely easy-to-use and easy-to-add-to-my-cmake-project library for CDTs. from what I've read it must be one hell of a task to implement this. My preliminary tests all worked flawlessly.

I'm currently looking into using this library for various steps in fitting NURBS surfaces to meshes, which might be a surprising usage, but both the parameterization step, as well as "cutting" the spline surface and remeshing it benefit vastly from a good CDT library.

My question concerns the only real C alternative that I would consider, which is John Shewchuck's "triangle" library (https://www.cs.cmu.edu/~quake/triangle.html): How well do you know it, and how well does your library compare to it? "Triangle" provides quite a lot of options, which is not always a good thing, since your library just seems to "just work", but I'm wondering if there are things that "Triangle" does that this library can't (as of yet). 

At the moment I'm trying to make a wrapper that emulates "Triangle"'s i/o structure so that I can use your library for the parts of libigl (https://github.com/libigl/libigl) that rely on "Triangle" (the reason being licensing issues with "Triangle"), and for now it seems, that the way to go is to ignore almost all of the command line options.

Thanks again for this wonderful project!

**Reply by artem-ogre** [reply]
Hi, Christopher! 
Thank you very much for the kind words: this made my day :)
I've created [a new discussion](https://github.com/artem-ogre/CDT/discussions/29) for this topic.

**Reply by artem-ogre** [reply]
@RotateMe I answered in that other discussion
